easybuild/easyconfigs/w/wheel/wheel-0.29.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'PythonPackage'
+
+name = 'wheel'
+version = '0.29.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/wheel'
+description = """A built-package format for Python."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+]
+
```

Comment:
@FokkeDijkstra I would include a custom `sanity_check_paths` in here (& the other `wheel` easyconfig):

```python
sanity_check_paths = {
    'files': ['bin/wheel'],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```